BeeSwift/GoalViewController.swift

```diff
@@ -515,7 +528,10 @@ class GoalViewController: UIViewController,  UIScrollViewDelegate, DatapointTabl
     func updateInterfaceToMatchGoal() {
         self.datapointTableController.hhmmformat = goal.hhmmFormat
         self.datapointTableController.datapoints = goal.recentData.sorted(by: {$0.updatedAt < $1.updatedAt})
-
+        
+        self.deltasLabel.isHidden = goal.dueByTable.entries.isEmpty
```

Comment:
[Nit] This won't collapse the height - maybe an argument for StackView here as well. Doesn't seem like a big deal though.

Reply:
I would hope the app's UI would all be written in `SwiftUI` before long. Shall the views first be migrated to `UIKit`'s `UIStackView`?